doc/users/whats_new/style_changes.rst

```diff
@@ -0,0 +1,98 @@
+Changes to the default style
+----------------------------
+
+The most important changes in matplotlib 2.0 are the changes to the
+default style.
+
+While it is impossible to select the best default for all cases, these
+are designed to work well in the most common cases.
+
+These changes include:
+
+Colors
+``````
+
+- The default figure background color has changed from grey to white.
```

Comment:
I see this change in the template, but not in `rcsetup.py`; is it fully implemented?